tests/basic_one_way_send.go

```diff
@@ -0,0 +1,175 @@
+package tests
+
+import (
+	"context"
+	"math/big"
+	"time"
+
+	avalancheWarp "github.com/ava-labs/avalanchego/vms/platformvm/warp"
+	warpBackend "github.com/ava-labs/subnet-evm/warp"
+	. "github.com/ava-labs/teleporter/tests/utils"
+
+	"github.com/ava-labs/awm-relayer/messages/teleporter"
+	"github.com/ava-labs/subnet-evm/core/types"
+	"github.com/ava-labs/subnet-evm/interfaces"
+	"github.com/ava-labs/subnet-evm/params"
+	"github.com/ava-labs/subnet-evm/x/warp"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	teleporterMessage = teleporter.TeleporterMessage{
+		MessageID:               big.NewInt(1),
+		SenderAddress:           FundedAddress,
+		DestinationAddress:      FundedAddress,
+		RequiredGasLimit:        big.NewInt(1),
+		AllowedRelayerAddresses: []common.Address{},
+		Receipts:                []teleporter.TeleporterMessageReceipt{},
+		Message:                 []byte{1, 2, 3, 4},
+	}
+)
+
+// Ginkgo describe node that acts as a container for the teleporter e2e tests. This test suite
+// will run through the following steps in order:
+// 1. Send a transaction to the Teleporter contract on Subnet A
+// 2. Aggregate signatures and send the Warp message to Subnet B
+// 3. Verify receipt of the message on Subnet B
+var _ = ginkgo.Describe("[Teleporter one way send]", ginkgo.Ordered, func() {
```

Comment:
High level question:

Is the idea that each of the current "integration test" scripts will be translated into its own `ginkgo.Describe` container? It seems like, alternatively, each integration test script could be translated into its own `It` within a single `ginkgo.Describe` container, such that the `It`s should then be able to run in parallel.

Reply:
Reading the Ginkgo docs again, the latter approach seems to be what we should aim for. For our use case, the `Describe` container should roughly translate to: "Describe the specs that should be met when messages are passed between subnets using Teleporter". Then, the `It` containers should roughly translate to each individual integration test. I agree we should refactor this code to reflect this.

Reply:
I refactored the tests to follow the pattern described above. It was relatively straightforward, and no major framework changes needed to be made.

Reply:
🙌